service/src/main/resources/application.yml

```diff
@@ -57,6 +57,15 @@ drshub:
         - type: https
           auth: current_request
           fetchAccessUrl: true # Used for Azure
+    sage:
+      name: Sage Bionetworks
+      hostRegex: 'repo-([^.]+)\.\1\.sagebase\.org' # Dev: 'repo-dev.dev.sagebase.org' & Prod: 'repo-prod.prod.sagebase.org'
+      metadataAuth: false
+      ecmFenceProvider: SAGE
+      accessMethodConfigs:
+        - type: s3
+          auth: fence_token
```

Comment:
This is confusing because this has sage does not use fence. This should probably be changed to `provider_token` and all references to `fence` be updated to be `provider`. But that is probably out of scope for this ticket.

Reply:
Is this for all the providers or just sage?